miqa/core/models/annotation.py

```diff
@@ -0,0 +1,42 @@
+from enum import Enum
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils import timezone
+
+
+class Decision(Enum):
+    NONE = '-'
```

Comment:
I guess we still need an explicit `NONE`, if only for the case where an annotation was added, but then someone wishes to explicitly unset it.